client/components/domains/search-filters/dropdown-filters.jsx

```diff
@@ -65,32 +61,10 @@ export class DropdownFilters extends Component {
 	getFiltercounts() {
 		return (
 			( this.props.lastFilters.tlds?.length || 0 ) +
-			( this.props.lastFilters.exactSldMatchesOnly && 1 ) +
-			( this.props.lastFilters.maxCharacters !== '' && 1 )
+			( this.props.lastFilters.exactSldMatchesOnly && 1 )
 		);
 	}
 
-	getMaxCharactersValidationErrors() {
-		const {
-			filters: { maxCharacters },
-			translate,
-		} = this.props;
-		const isValid = /^-?\d*$/.test( maxCharacters );
-		return ! isValid ? [ translate( 'Value must be a whole number' ) ] : null;
-	}
-
-	getOverallValidationErrors() {
-		const isValid = this.getMaxCharactersValidationErrors() === null;
-		const { showOverallValidationError } = this.state;
-		return ! isValid && showOverallValidationError
-			? [ this.props.translate( 'Please correct any errors above' ) ]
-			: null;
-	}
```

Comment:
Since `getMaxCharactersValidationErrors` only returned `null`, `isValid` was always `true` and this method always returned `null`, so it can be removed too.